src/v/kafka/protocol/schemata/generator.py

```diff
@@ -436,6 +436,27 @@ def make_context_field(path):
     "FinalizedFeatureKey",
 ]
 
+# a list of struct types which are ineligible to have default-generated
+# `operator==()`, because one or more of its member variables cannot do
+# three-way comparsion.
```

Comment:
instead of `three-way` you can say `are not comparable`

Reply:
sure.